src/Common/src/System/Net/Logging/GlobalLog.cs

```diff
@@ -253,12 +253,17 @@ public static void Dump(byte[] buffer, int offset, int length)
                 return;
             }
 
+            if (length == 0)
+            {
+                EventSourceLogging.Log.DebugMessage("lenth of ArraySegment in Dump is 0");
+                return;
+            }
+
             var bufferSegment = new byte[length];
             for (int i = 0; i < length; i++)
             {
                 bufferSegment[i] = buffer[offset + i];
             }
-
```

Comment:
Should have a blank line after the for braced clause and before the EventSource* line.